src/docs/product/accounts/quotas/spike-protection.mdx

```diff
@@ -7,9 +7,13 @@ description: "Learn how to manage your quota and make sure that your volume isn'
 
 A spike is a significant, temporary increase in your error event volume. Because Sentry bills based on the number of monthly events sent, spikes can quickly consume your quota for an entire month. Spike Protection guards against this by establishing a spike threshold for the average number of error events each of your projects sends and then dropping them after that threshold is reached.
 
+<Note>
+Notifications for Spike Protection are turned off by default to avoid excessive noise for users. Read the [notification](/product/accounts/quotas/spike-protection/#notifications) section below to learn how to turn them on for key projects.
+</Note>
+
```

Comment:
```suggestion
<Note>
Notifications for Spike Protection are turned off by default to avoid excessive noise for users. Read the <a class href=“/product/accounts/quotas/spike-protection/#notifications”>notification</a> section below to learn how to turn them on for key projects.
</Note>

```